cmd/simulacra/simulacra.go

```diff
@@ -351,12 +351,16 @@ func runCustomScripts(ctx context.Context, vm *gce.VM, logger *logging.Directory
 	return nil
 }
 
-func getRecommendedMachineType(imageFamily string, image string) string {
-	if imageFamily != "" {
-		return agents.RecommendedMachineType(imageFamily)
+func constructImageSpec(config *Config) (string, error) {
+	// If both or neither set, return error
+	if (config.ImageFamily != "") == (config.Image != "") {
+		return "", return fmt.Errorf("config.ImageFamily (%v) or config.Image (%v) must be set, but not both", config.ImageFamily, config.Image)
```

Comment:
compile error. it should be easier to `go build` simulacra because it's a command line binary that produces an executable.

Reply:
Fixed